engine/runtime/src/test/java/org/enso/interpreter/test/MetaIsATest.java

```diff
@@ -12,16 +14,17 @@
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
-import org.junit.After;
-import org.junit.Before;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
 public class MetaIsATest extends TestBase {
-  private Context ctx;
-  private Value isACheck;
+  private static Context ctx;
+  private static Value isACheck;
+  private static ValuesGenerator generator;
 
-  @Before
-  public void prepareCtx() throws Exception {
+  @BeforeClass
+  public static void prepareCtx() throws Exception {
```

Comment:
This seems like a good idea. I did similar thing in many tests that I created - the context is created in `@BeforeClass`. Moreover, I followed the same pattern with my other tests where standalone nodes are tested - they are created in `@BeforeClass` method rather than in `@Before` method.